packages/pnpm/README.md

```diff
@@ -14,7 +14,7 @@
 
 Features:
 
-* **Fast.** As fast as npm and Yarn.
+* **Fast.** Almost always under 20s with cold cache and under 10s with hot cache ([benchmarks](https://github.com/pnpm/benchmarks-of-javascript-package-managers)).
```

Comment:
Not so sure I like this change. 10 or 20 seconds sounds slow without comparing to other package managers. I think it's better to say something like "Up to 2x faster than npm and Yarn." Then provide a link to benchmarks that prove it.